components/drone_mobile

```diff
@@ -0,0 +1,6 @@
+{	
+  "name": "drone_mobile",	
+  "owner": ["@bjhiltbrand"],	
+  "manifest": "https://github.com/bjhiltbrand/drone_mobile_home_assistant/blob/main/custom_components/drone_mobile/manifest.json",	
```

Comment:
This should be using the raw link, e.g., <https://raw.githubusercontent.com/bjhiltbrand/drone_mobile_home_assistant/main/custom_components/drone_mobile/manifest.json>